Dockerfile.tmpl

```diff
@@ -28,6 +28,9 @@ ENV PATH="~/.local/bin:${PATH}"
 # b/183041606#comment5: the Kaggle data proxy doesn't support these APIs. If the library is missing, it falls back to using a regular BigQuery query to fetch data.
 RUN uv pip uninstall --system google-cloud-bigquery-storage
 
+# b/394382016: sigstore (dependency of kagglehub) requires a prerelease packages, installing separate.
```

Comment:
I don't love that we're depending on a prerelease package inside our released version of Kagglehub, that feels like a bit of a breakage of versioning.

Thanks for filing a bug to move this back into requirements though, I'd like to avoid having too many exceptions.